index.html

```diff
@@ -191,5 +191,64 @@ <h5>Skills</h5>
         </li>
       </ul>
     </section>
+
+    <!-- contact-me -->
+    <section id="contact-form">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+
+      <form action="https://formspree.io/f/meqdoeyo" method="post">
```

Comment:
Your design of the input elements is not matching the Figma template of your choosing. In the Figma design, there is no border around input elements. Kindly update it accordingly. 
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/186372992-df6b7d9f-f2c1-4c32-aea3-dc0fa16435a3.png) | ![image](https://user-images.githubusercontent.com/33268927/186372891-25b128d3-0a80-47a2-9d4b-a02bdfb79cc4.png) |